webapp/src/templates/partials/reports_content.html

```diff
@@ -64,14 +64,16 @@
               {{field.label}}
               <span class="fa fa-cogs" ng-show="field.generated" translate translate-attr-title="Generated report field"></span>
             </label>
-            <span ng-if="field.hasUrl"><a ui-sref="contacts.detail({ id: reportsContentCtrl.summaries[$index].subject._id || reportsContentCtrl.summaries[$index].fields.patient_uuid })">{{field.value}}</a></span>
-            <span ng-if="!field.hasUrl">{{field.value}}</span>
+            <span ng-if="field.target && field.target.filter"><a ng-click="reportsContentCtrl.search(field.target.filter)">{{field.value}}</a></span>
+            <span ng-if="field.target && field.target.url"><a ui-sref="{{field.target.url.route}}({{field.target.url.params}})">{{field.value}}</a></span>
+            <span ng-if="!field.target">{{field.value}}</span>
```

Comment:
I _think_ this also fixes https://github.com/medic/cht-core/issues/6149
3 birds with one stone?

Reply:
Really? Cool! I'll investigate.

Reply:
Totally! Added to the PR description.